### ISSUE TITLE
Add an extra blank line after command / before prompt?

### ISSUE THREAD

**Issue by johnnykwwang** [bhilburn/powerlevel9k#403]
Hi, is there any way we can add a blank line **after an output** or **before a prompt**?

just like the bullet-train theme:
![image](https://cloud.githubusercontent.com/assets/13166286/22964243/862471b6-f392-11e6-8fee-8373b3400021.png)

**Comment by johnnykwwang** [comment]
Found a temporary solution, although kind of tricky.
Since I can't just use '\n' to prompt prefix as the trailing new line is deleted
I added a space to both line.
```
POWERLEVEL9K_MULTILINE_FIRST_PROMPT_PREFIX="\n "
POWERLEVEL9K_MULTILINE_SECOND_PROMPT_PREFIX=' $ '
```
It now looks like this, notice that there's a space before each line
![image](https://cloud.githubusercontent.com/assets/13166286/23008380/1c2db2f0-f44a-11e6-9d7a-26dd7da9a2f4.png)

But I still hope to find a real solution though, that whitespace is little bit annoying

**Comment by thuandt** [comment]
@johnnykwwang my queston isn't related with your issue, but can you share fonts and config of your prompt  which can display beautiful battery icon

**Comment by johnnykwwang** [comment]
@thuandt i didn't do anything about that battery icon, maybe it's in the font i use,
![image](https://cloud.githubusercontent.com/assets/13166286/23094877/b6ac9704-f63b-11e6-9890-948b0f9cf359.png)

 but for better icon support I end up installing[ gabrielelana/awesome-terminal-fonts ](https://github.com/gabrielelana/awesome-terminal-fonts/) and now it looks even more awesome.  Maybe I'll post my final outcome later.

**Comment by rjorgenson** [comment]
@thuandt That battery icon is the fallback standard UTF-8(I think?) battery icon, it should be in every font and is 🔋 

You can override the icon for your icon settings if you want to use just that battery icon by setting `POWERLEVEL9K_BATTERY_ICON="\U1F50B"` in your .zshrc

**Comment by bhilburn** [comment]
@johnnykwwang - Hey! I'm actually not sure I understand the exact issue in the latest screenshot you posted (clever work-around, by the way!).

Can you add an arrow the image specifically calling out the whitespace you want to get rid of?

**Comment by rjorgenson** [comment]
@bhilburn johnnykwwang already figured the whitespace out, see #408 =]

**Comment by bhilburn** [comment]
Ah! Got it. Thanks, @rjorgenson =)

**Comment by bhilburn** [comment]
Merged #408 =)
